docs(tutorials): updated Kubeflow tutorial MTA-1753 (#403)
scaleway/docs-content: tutorials/kubeflow-on-kapsule/index.mdx

    - **Choose your cluster type**: Choose a Kubernetes Kapsule cluster.
    - **Choose a region**: Choose the region in which you want your cluster to be created. A region refers to the geographical location in which your cluster will be created.
    - **Choose a Kubernetes Version**: Select the Kubernetes version for the cluster. Choose the latest version from the drop-down list.
    - **Enter a Name for the Cluster**: Name your cluster or leave the automatically generated name. 
    - Keep the default values in the `Advanced Options` section.
    - Click **Next** to configure your pool.
    - **Choose a zone**: An [Availability Zone (AZ)](/compute/instances/concepts/#availability-zone) refers to the geographical location in which all your pool's nodes will be created.
    - **Select a node type**: Choose the node type. We will be using `GP1-M` Instances for this tutorial (16 cores, 64 GB RAM).
    - **Configure node options**:
      - Enable the autoscale feature.
      - Enter the minimum and maximum number of nodes available for the pool. Choose a minimum of **1 node** and a maximum of **2 nodes**. 

<Message type="note">

>  In case you choose an under dimensioned configuration for your base cluster, and if you later add GPU nodes to your pool with autoscaling, the automatic scaling down of the GPU nodes might be prevented by Kubeflow system pods being deployed on the GPU nodes (due to the high cpu/memory pressure on the CPU nodes).

Reviewer: ```suggestion
 If you choose an under-dimensioned configuration for your base cluster, and you later add GPU nodes to your pool with autoscaling, you may have problems with automatic downscaling of the GPU nodes. This can happen when  Kubeflow system pods deployed on the GPU nodes prevent downscaling, due to the high CPU/memory pressure on the CPU nodes.
```

> 1. Log in into your [Scaleway Console](https://console.scaleway.com). The Console dashboard displays.

Author: ```suggestion
1. Log in into your [account](https://console.scaleway.com). 
```

> 1. Log in into your [Scaleway Console](https://console.scaleway.com). The Console dashboard displays.

Author: I think adding the link to the how to create a cluster would impact the flow as there are specific configurations that are needed for this tuto.

## Configuring the connection to your cluster

> A `.kubeconfig` has been generated during your cluster's creation. Download this file to manage your Kubernetes Kapsule cluster using the `kubectl` command-line tool from your local computer.

Reviewer 2: ```suggestion
A `.kubeconfig` file is generated during your cluster's creation. Download the file to manage your Kubernetes Kapsule cluster using the `kubectl` command-line tool from your local computer.
```

> 1. Download your `.kubeconfig` file by clicking on **Download file** on the cluster's information page:

Reviewer 2: ```suggestion
1. Click on **Download file** to download your `.kubeconfig` file in the cluster information page:
```
